modules/bridge/src/main/ts/ephox/bridge/components/toolbar/ToolbarLabel.ts

```diff
@@ -0,0 +1,50 @@
+import { StructureSchema } from '@ephox/boulder';
+import { Optional, Result } from '@ephox/katamari';
+
+import * as ComponentSchema from '../../core/ComponentSchema';
+
+export interface BaseToolbarLabelSpec<I extends BaseToolbarLabelInstanceApi> {
+  tooltip?: string;
+  icon?: string;
+  text?: string;
+  onSetup?: (api: I) => (api: I) => void;
```

Comment:
Not sure we need a `onSetup` if there is no API we can add that later if we need an api.